index.js

```diff
@@ -285,17 +285,41 @@ export function escapePath(path) {
 	return path.replace(/[\\.[]/g, '\\$&');
 }
 
+// The keys returned by Object.entries() for arrays are strings
+function entries(value) {
+	if (Array.isArray(value)) {
+		return value.map((value, index) => [index, value]);
```

Comment:
I _think_ you can `Object.entries()` on an array to get the same thing, so you may not need this check.

Reply:
No

```js
console.log(Object.entries(1));
//=> ['0', 1]
```